src/types/field/fieldTypes.ts

```diff
@@ -48,7 +48,7 @@ export enum MyInfoAttribute {
 }
 
 export enum SPCPFieldTitle {
-  SpNric = 'SingPass Validated NRIC',
-  CpUid = 'CorpPass Validated UID',
-  CpUen = 'CorpPass Validated UEN',
+  SpNric = 'Singpass Validated NRIC',
```

Comment:
@mantariksh correct me if i'm wrong, this will affect RPA right? Since email JSONs will change their field titles. 

@syan-syan fyi